### PR TITLE
adding a list of comments to singleReview page

### PR DESCRIPTION
**Pull request by kyrikoni**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -15,3 +15,10 @@ export const getSingleReview = (review_id) => {
     return res.data.review;
   });
 };
+
+export const getReviewComments = (review_id) => {
+  return gamesApi.get(`/api/reviews/${review_id}/comments`).then((res) => {
+    console.log(res.data.comments);
```
Comment (B7Bprog) [inline src/api.js]: Please remove logs.

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+import { useEffect, useState } from "react";
+
 export const Comments = () => {
-  return <p>Comments Go Here</p>;
+  const [comments, setComments] = useState([]);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    getReviewComments(review_id).then((comments) => {
+      setComments(comments);
```
Comment (B7Bprog) [inline src/components/Comments.jsx:13]: Please use a 'loading' state to give a visual feedback of loading comments to the user.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Kyri, please see my comments, amend your code and then feel free to merge it.

**Review by Pipe-Smoking-Rabbit — APPROVED** [review]
Small feedback on slack, otherwise good to merge!

**Pull request merged**